Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -743,22 +917,22 @@ fileprivate extension Compiler.ByteCodeGen {
         guard let val = c._singleScalarAsciiValue else {
           return false
         }
-        builder.buildQuantify(asciiChar: val, kind, minTrips, maxExtraTrips, isScalarSemantics: isScalarSemantics)
+        builder.buildReverseQuantify(asciiChar: val, kind, minTrips, maxExtraTrips, isScalarSemantics: isScalarSemantics)
```

Comment:
Should these be guarded by an `if reverse`? (With the diff I can't quite see the full logic here).

Reply:
Nope, this is part of `tryEmitFastReverseQuant`. The diff is showing `tryEmitFastQuant` as an addition and the reverse variant as a change to the original